### ISSUE TITLE
Browser (Chrome) hanging when closing tabs with Live TL (Youtube or Live TL own page)

### ISSUE THREAD

**Issue by gakiloroth** [LiveTL/LiveTL#216]
**Describe the bug**
My browser will sometimes hang when closing YouTube / Live TL tabs. I suspected it was potentially the HyperChat, but even with both on and off this would still happen. It would freeze and show the loading cursor and sometimes be fine in 5-10 seconds, sometimes a minute or so.

**To Reproduce**
1. Go to any stream with a chat
2. Leave it on for a while (usually 30mins + was enough for this to happen for me, with "Live chat" - since this doesn't happen all the time I suspect very active chats maybe causing this issue but I am not sure)
3. Attempt to close the tab
4. Browser should hang (freezes for a couple seconds up to a minute+)

**Expected behavior**
Closing the tab would not cause browser to hang

**Image**
This is me middle-clicking on a Youtube tab to close it with theatre mode and Live TL on after watching a stream
![image](https://user-images.githubusercontent.com/9069768/119583384-5e8f2a80-bd7b-11eb-8aaa-6f11b9741ee6.png)

Another example but on LiveTL page 
![image](https://user-images.githubusercontent.com/9069768/119612276-8c429680-bdb0-11eb-94e8-daa5d9c471d0.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Chrome
 - Version: 90.0.4430.212

**Additional context**
I tested this with HyperChat on and off, and then with disabling Live TL completely. With Live TL off, it didn't happen. Its possible this is some other incompatibility, I will continue to try and mess around with settings / Live TL on or off.

**Comment by KentoNishi** [comment]
weird, will look into it

**Comment by gakiloroth** [comment]
A note, this only started happening recently (I believe after the Kanatran notice on reddit but my memory isn't 100% sure)

Basically just saying this hasn't been around for the majority of my use of extension

**Comment by KentoNishi** [comment]
do you remember if this was introduced in v6 or not? since v5 and v6 are completely different extensions

**Comment by gakiloroth** [comment]
Ah I can't be sure... I'm not sure if there's a way to check when my extension was updated. My mental estimation of this happening is within 2-4 weeks in the past, but that may not be very helpful 😓

**Comment by Hughp135** [comment]
question that is potentially relevant, but did you have the browser tab active the whole time you were watching the stream? (or at least within the last minute of you closing it)?

it could be caused by too much code going on in the background, which performs operations that are paused by chrome when the tab is in the background (just speculating though, so this isn't very useful)

**Comment by gakiloroth** [comment]
Yea it happens on tabs that I keep open but not in focus

**Comment by KentoNishi** [comment]
@gakiloroth quick question, does this still happen on the newest version?

**Comment by r2dev2** [comment]
I couldn't replicate on chromium under arch.

**Comment by gakiloroth** [comment]
I managed to recreate it leaving a stream with HC on and on Live Chat for a couple hours (on YouTube native site), but the freeze was much shorter than what I had experienced in the past. Sometimes I don't see this "loading wheel" + white screen at all and it only freezes for a couple seconds. 

In summary, could still recreate slight freeze, but it is much shorter than before.

![image](https://user-images.githubusercontent.com/9069768/125152136-7f2efc00-e0ff-11eb-89a2-84cd57d7f3be.png)

done on this stream by Amelia Watson: https://www.youtube.com/watch?v=rWWyO4eZ2dw (not sure if this info would do anything but linking anyways)
